CMSgov/managed-care-review
# [MR-412] State can indicate rate per cell or rate ranges

## Summary
[MR-412](https://qmacbis.atlassian.net/browse/MR-412)
State can indicate if utilizing rates per rate cell or rate ranges by selecting from a new radio group on the `Rate Details` page

#### Related issues

#### Screenshots
<img src="https://user-images.githubusercontent.com/98117700/166948676-742a52c8-f09d-41d4-a70f-75cd793e6485.png" width="300"/>

<img src="https://user-images.githubusercontent.com/98117700/166948814-52dbd49a-31d2-425f-8c2b-275daaa815b1.png" width="300"/>

<img src="https://user-images.githubusercontent.com/98117700/166948930-cba581d8-b1d6-4258-877a-539d4419e06c.png" width="300"/>

#### Test cases covered
- `RateDetailsSummarySection.test.tsx`
   - `can render all rate details fields for amendment to prior rate certification submission`
      - Add check for rate capitation type display
- `RateDetails.test.tsx`
   - `loads with empty rate type and document upload fields visible`
      - Add check for rate capitation radio selections.
   - `cannot continue without selecting rate capitation type` [NEW]
   - `cannot continue if no documents are added`
      - Add code to select a rate capitation type.
- `stateSubmissionFormCommands`
   - `fillOutNewRateCertification` & `fillOutAmendmentToPriorRateCertification`
      - Update commands to click on a rate capitation type radio button.



## QA guidance
